config/config.go

```diff
@@ -198,6 +203,130 @@ type HTTPRewriteResponses struct {
 	RemoveHeaders          []HeaderNameValue `yaml:"remove_headers,omitempty"`
 }
 
+// VerifyClientCertificateMetadata defines verification rules for client certificates, which allow additional checks
```

Comment:
We don't typically have such verbose comments in runtime code, but given the development on this codebase is slowing down and is relatively stable, I think it is helpful and I like it.